WordPress/press-this
# Restore bookmarklet feature

Fixes #15 

Testing instructions:

* Verify install instructions are present at wp-admin/tools.php
* Install bookmarklet
* Test pressing a page

(Please restrict comments on this PR to issues with the code present and implementation. General commentary about restoring the bookmarklet should continue on #15).
